tests/e2e/test_rosacli_cluster.go

```diff
@@ -303,3 +308,230 @@ var _ = Describe("Edit cluster",
+					"--replicas":             k,
+				})
+				stdout, err := rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
+				Expect(err).NotTo(BeNil())
+				Expect(stdout.String()).To(ContainSubstring(v))
+			}
+			if rosalCommand.CheckFlagExist("--multi-az") {
+				invalidReplicasErrorMapMultiAZ := map[string]string{
+					"2":  "Multi AZ cluster requires at least 3 compute nodes",
+					"0":  "Multi AZ cluster requires at least 3 compute nodes",
+					"-3": "must be non-negative",
+					"5":  "Multi AZ clusters require that the number of compute nodes be a multiple of 3",
+				}
+				for k, v := range invalidReplicasErrorMapMultiAZ {
+					rosalCommand.ReplaceFlagValue(map[string]string{
+						"--replicas": k,
+					})
+					stdout, err := rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
+					Expect(err).NotTo(BeNil())
+					Expect(stdout.String()).To(ContainSubstring(v))
+				}
+			} else {
+				invalidReplicasErrorMapSingeAZ := map[string]string{
+					"1":  "requires at least 2 compute nodes",
+					"0":  "requires at least 2 compute nodes",
+					"-1": "must be non-negative",
+				}
+				for k, v := range invalidReplicasErrorMapSingeAZ {
+					rosalCommand.ReplaceFlagValue(map[string]string{
+						"--replicas": k,
+					})
+					stdout, err := rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
+					Expect(err).NotTo(BeNil())
+					Expect(stdout.String()).To(ContainSubstring(v))
+				}
+			}
+
+			By("Check the validation for region")
+			invalidRegion := "not-exist-region"
+			rosalCommand.ReplaceFlagValue(map[string]string{
+				"--region":   invalidRegion,
+				"--replicas": originalReplicas,
+			})
+			stdout, err = rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
+			Expect(err).NotTo(BeNil())
+			Expect(stdout.String()).To(ContainSubstring("Unsupported region"))
+
+			By("Check the validation for billing-account for classic sts cluster")
+			rosalCommand.ReplaceFlagValue(map[string]string{
+				"--region": originalRegion,
+			})
+			rosalCommand.AddFlags("--billing-account", "123456789")
+			stdout, err = rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
+			Expect(err).NotTo(BeNil())
+			Expect(stdout.String()).To(ContainSubstring("Billing accounts are only supported for Hosted Control Plane clusters"))
+		})
+
+		It("to validate to create the sts cluster with the version not compatible with the role version	- [id:45176]", labels.Medium, func() {
+			clusterService := rosaClient.Cluster
+			accrolePrefix := "testAr45176"
+			path := "/a/b/"
+			ocmResourceService := rosaClient.OCMResource
+			clusterName := "cluster45176"
+			operatorRolePrefix := "cluster45176-xvfa"
+			By("Create account-roles in low version 4.14")
+			output, err := ocmResourceService.CreateAccountRole("--mode", "auto",
+				"--prefix", accrolePrefix,
+				"--path", path,
+				"--version", "4.14",
```

Comment:
Hardcoding the version means we need to update it in the future. It is better to get the lower version dynamically.